tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -68,6 +70,49 @@ export const getNumAlerts = createSelector(
   }
 );
 
+export const getAlertsFocusType = createSelector(
+  selectDebuggerState,
+  (state: DebuggerState): AlertType | null => {
+    return state.alerts.focusType;
+  }
+);
+
+/**
+ * Get number of alerts of the alert type being focused on.
+ *
+ * If no alert type focus exists, returns 0.
+ * The returned number is regardless of whether the detailed Alerts
+ * data have been loaded by the front end.
+ */
+export const getNumAlertsOfFocusedType = createSelector(
+  selectDebuggerState,
+  (state: DebuggerState): number => {
+    if (state.alerts.focusType === null) {
+      return 0;
+    }
+    return state.alerts.alertsBreakdown[state.alerts.focusType] || 0;
```

Comment:
No functional difference right now but you probably mean `return state.alerts.alertsBreakdown[state.alerts.focusType] ?? 0;` I am not sure if TypeScript we use supports nullish-coalescing.